code/datums/mutations/touch.dm

```diff
@@ -49,3 +54,46 @@
 	else
 		to_chat(user,"<span class='warning'>The electricity doesn't seem to affect [target]...</span>")
 		return ..()
+
+#define HALFWAYCRITDEATH ((HEALTH_THRESHOLD_CRIT + HEALTH_THRESHOLD_DEAD) * 0.5)
```

Comment:
```suggestion

```

Reply:
note that HALFWAYCRITDEATH gets undefined at the end of the file for defib code, so maybe you should just move its definition to a different file or something?

Reply:
Yep ! It's why the linter is failing, I'll fix when I'm home.

Reply:
yes, move the definition